main.tf

```diff
@@ -85,23 +87,24 @@ module "loadbalancer" {
   subnetwork   = local.subnetwork
   region       = var.region
   instances    = module.instances.compilers
-  has_lb       = local.has_lb
+  has_lb       = local.has_lb && local.create_network
```

Comment:
@jarretlavallee Why did you choose to disable load balancer creation when re-using existing networks?

Reply:
The complexity of configuring a  load balancer for an external or existing network seemed like too much for this module. So I made the assumption that if you were providing existing networking, you would provide load balancers as well.

In my use case, we are using a subproject network that is shared with our project. That network would require a specific configuration in order to configure load balancers that would likely not apply to others that use this code.

Should I make it an optional item with its own variable instead of making the condition on creating networks?